### PR TITLE
Research

### PR DESCRIPTION
**Pull request by danyellz**
@Bontgoy  it looks like explore.js is running properly in /js, but undefined is being outputted along with the reverse string. Any idea what's going on?

### CODE REVIEW

File: js/explore.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//Create the ability to input a word or sentence then output it in reverse. 
+// Do this by reading the word from the last character. Reading the input in reverse, 
+//take each letter and save into a new STRING, forming a new word. 
+//Printing this new word will show the initial word in reverse.
+
+function Reverse(string){
```
Comment (Bontgoy) [inline js/explore.js:6]: caps `Reverse` should be downcase for a function `reverse`

don't name a function with caps.  Check jS syntax guidle lines if you have any questions

File: js/explore.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	char = '';
+	for (var i = string.length; i > 0; i--)
+		char += string[i];
+		return char;
```
Comment (Bontgoy) [inline js/explore.js:10]: once a function sees `return` it stops running and returns the request.  This means you're returning the first thing that runs here.

File: js/explore.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	}
+
+Reverse("This is my super short string")
+
+if (char.length < 100) {
```
Comment (Bontgoy) [inline js/explore.js:15]: `chars` is a var that you created inside your function.  This outside of the function hence the undefined

File: js/explore.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+//Printing this new word will show the initial word in reverse.
+
+function Reverse(string){
+	char = '';
+	for (var i = string.length; i > 0; i--)
+		char += string[i];
+		return char;
+	}
+
+Reverse("This is my super short string")
```
Comment (Bontgoy) [inline js/explore.js:13]: you should be logging the results to this

File: js/explore.js (hunk 3 above)
Comment (danyellz) [inline js/explore.js:15]: Hey Tyler, thanks for the feeback. That makes sense and was my initial thought. However, if if I make the char variable inaccessible from outside of the function I can't store it's value. Right?

Comment (danyellz) [inline js/explore.js:15]: For example if I declare char as global, the function will return undefinedgnirts trohs repus ym si sih. If I set it's scope to the function I'll get an error.

**Pull request merged**